cppcache/src/ClientProxyMembershipID.cpp

```diff
@@ -77,21 +77,13 @@ ClientProxyMembershipID::ClientProxyMembershipID(
 }
 
 void ClientProxyMembershipID::initHostAddressVector(
-    const ACE_INET_Addr& address) {
-  if (address.get_type() == AF_INET6) {
-    const auto socketAddress6 =
-        static_cast<const struct sockaddr_in6*>(address.get_addr());
-    auto socketAddress =
-        reinterpret_cast<const uint8_t*>(&socketAddress6->sin6_addr);
-    auto length = sizeof(socketAddress6->sin6_addr);
-    m_hostAddr.assign(socketAddress, socketAddress + length);
+    const boost::asio::ip::address& address) {
+  if (address.is_v6()) {
+    auto bytes = address.to_v6().to_bytes();
+    m_hostAddr.assign(bytes.begin(), bytes.end());
```

Comment:
This is probably as good a time as any to rename member variables...

Reply:
Good point!